python/lsst/ts/tcpip/mock_json_reading_one_client_server.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of ts_tcpip.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+import typing
+from collections import deque
+
+from .constants import LOCAL_HOST
+from .json_reading_one_client_server import JsonReadingOneClientServer
+
+__all__ = ["MockJsonReadingOneClientServer"]
+
+
+class MockJsonReadingOneClientServer(JsonReadingOneClientServer):
```

Comment:
I worry that this name will confusing users by making them think they should use it for mocking low-level controllers.

I suggest a name such as `TrivialJsonOneClientServer`, or perhaps `Test` instead of `Trivial`. If you go with `Test` please name the module `testjson...` instead of `test_json...` so pytest won't try to run it as a unit test. And also check that pytest doesn't try to run it based on the class name.

Reply:
I settled for `Test`. I named the files as you suggested and pytest is happy with them.